config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -31,3 +32,120 @@ presubmits:
               - run
               - k8s.io/publishing-bot/cmd/validate-rules
               - /home/prow/go/src/k8s.io/kubernetes/staging/publishing/rules.yaml
+
+  kubernetes/kubernetes:
+    - name: pull-publishing-bot-dry-run
+      always_run: false
+      decorate: true
+      path_alias: k8s.io/kubernetes
+      skip_report: false
+      optional: true
+      run_if_changed: '^staging/publishing.*$'
+      extra_refs:
+        - org: kubernetes
```

Comment:
Here's a problem, what happens when we need to add a new staging repo? will this job break during the process? (and hence not allow the PR with new staging repo to merge?)

could we pull the other repos using a script?

Reply:
Also we need the job to have way more memory to run (and it needs a volume too ... not sure how that will work)